sfm_pc/management/commands/import_google_doc.py

```diff
@@ -221,6 +243,17 @@ def handle(self, *args, **options):
                             self.current_row = one_index_start + (index + 1)
                             getattr(self, 'create_{}'.format(entity_type))(row)
 
+        # Check the entity maps after bringing in all data, because there are
+        # references to organizations and people in multiple sheets.
+        for entity_type in ('organization', 'person'):
```

Comment:
why do you want to log these errors late instead of at the time of the potential error is found?

Reply:
We won't have a complete mapping of entities until all of the data is brought in. We may be able to recognize a conflict during import, but we want to log a message for all records that reference that entity, which we can't do until the import is complete.

Reply:
are you saying that you want to have the log messages unified or that you can't know if there's a problematic dupe until you have all the records?

this first seems true, but i don't understand the second if that's what you are saying.

Reply:
I want to have the log messages unified.